test/functionalTest/cases/0997_GET_v2_entity_single_attribute_value/GET_v2_entity_single_attribute_value.test

```diff
@@ -180,65 +180,41 @@ Date: REGEX(.*)
 02. GET /v2/entities/8787GHY/attrs/brand/value
 ==============================================
 HTTP/1.1 200 OK
-Content-Length: 61
-Content-Type: application/json
+Content-Length: 13
+Content-Type: text/plain
 Date: REGEX(.*)
 
-{
-    "value": {
-        "metadata": {},
-        "type": null,
-        "value": "Mercedes Benz"
-    }
-}
+Mercedes Benz
```

Comment:
Should this be

```
Mercedes Benz
```

or

```
"Merces Benz"
```

?

In order to kep "symmetry" with PUT operation, it should have "". In order to keep simpler, it should be without "".

I don't have an answer for this :) Please, cast the question in the NGSIv2 skype chat spec.